core/src/main/java/org/apache/carbondata/core/metadata/schema/PartitionInfo.java

```diff
@@ -17,17 +17,21 @@
 
 package org.apache.carbondata.core.metadata.schema;
 
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
 import org.apache.carbondata.core.metadata.schema.partition.PartitionType;
+import org.apache.carbondata.core.metadata.schema.table.Writable;
 import org.apache.carbondata.core.metadata.schema.table.column.ColumnSchema;
 
 /**
  * Partition information of carbon partition table
  */
-public class PartitionInfo implements Serializable {
+public class PartitionInfo implements Serializable, Writable {
```

Comment:
This modification will fix a bug : PartitionInfo is null in new CarbonTable in getSplits function in CarbonTableInputFormat. Could you split it into a independent PR? Looks inappropriate with the description of current PR.